src/main/java/core/basesyntax/service/impl/ReportServiceImpl.java

```diff
@@ -0,0 +1,28 @@
+package core.basesyntax.service.impl;
+
+import core.basesyntax.dao.StorageDao;
+import core.basesyntax.service.ReportService;
+import java.util.stream.Collectors;
+
+public class ReportServiceImpl implements ReportService {
```

Comment:
```suggestion
public class CsvReportServiceImpl implements ReportService {
```

Reply:
here you actually have specific logic for csv format, thats why lets name this service CsvReportServiceImpl